### ISSUE TITLE
Workarounds for lack of #5669

### ISSUE THREAD

**Issue by SwooshyCueb** [irods/irods#5828]
Until #5669 is implemented, the irods CMake import targets will not properly pass down all their dependencies. Targets building against them must themselves track these transient dependencies in the meantime.

**Comment by alanking** [comment]
Just noting that #5669 is now implemented.

**Comment by SwooshyCueb** [comment]
Yes and no. The issue is closed as there has been significant work towards it, but due to how we handle issue milestones, it was closed and split out into other issues for the remaining work.
